locust-simulations/scripts/run_tests.sh

```
@@ -0,0 +1,55 @@
+# HOST: target host URL (default: http://localhost:8082 or from env var)
+# SHAPE: load shape to use (stages, spike, multiple_spikes, ramping)
+
+MODE=${1:-ui}
+USERS=${2:-10}
+SPAWN_RATE=${3:-10}
+DURATION=${4:-300}
+HOST=${PRODUCT_SERVICE_URL:-${5:-http://localhost:8082}}
+SHAPE=${6:-""}
+
+# Move to the directory containing this script
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)